Dockerfile

```diff
@@ -1,46 +1,50 @@
-FROM alpine:3.7
+FROM alpine:3.6
 
-ENV BANNER "geodesic"
-
-# Where to store state
-ENV CACHE_PATH=/localhost/.geodesic
-
-ENV GEODESIC_PATH=/usr/local/include/toolbox
-ENV MOTD_URL=http://geodesic.sh/motd
-ENV HOME=/conf
-ENV KOPS_CLUSTER_NAME=example.foo.bar
+ONBUILD ARG BANNER="geodesic"
 
 # Install all packages as root
 USER root
 
+RUN echo -e "http://nl.alpinelinux.org/alpine/v3.6/main\nhttp://nl.alpinelinux.org/alpine/v3.6/community" > /etc/apk/repositories
 # Install common packages
-RUN apk update \
-    && apk add unzip curl tar \
-          python make bash vim jq figlet \
+RUN apk add --no-cache --update unzip curl tar \
+          python make jq figlet \
           openssl openssh-client sshpass iputils drill \
           gcc libffi-dev python-dev musl-dev openssl-dev py-pip py-virtualenv \
           git coreutils less groff bash-completion \
           fuse syslog-ng libc6-compat && \
+          rm -rf /tmp/* /var/cache/apk/* && \
           mkdir -p /etc/bash_completion.d/ /etc/profile.d/ \
     && mkdir -p /conf \
-    && touch /conf/.gitconfig \
+    && touch /conf/.gitconfig
 
 RUN echo "net.ipv6.conf.all.disable_ipv6=0" > /etc/sysctl.d/00-ipv6.conf
 
+# Where to store state
+ENV CACHE_PATH=/localhost/.geodesic
+
+ENV GEODESIC_PATH=/usr/local/include/toolbox
+ENV MOTD_URL=http://geodesic.sh/motd
+ENV HOME=/conf
+ENV KOPS_CLUSTER_NAME=example.foo.bar
+ENV SECRETS_PATH=${HOME}
+
 # Disable vim from reating a swapfile (incompatible with goofys)
-RUN echo 'set noswapfile' >> /etc/vim/vimrc
+#RUN echo 'set noswapfile' >> /etc/vim/vimrc
```

Comment:
the reason for this is if you `vim` a file on `goofys` filesystem, the `.swp` file generated by vim will cause problems.